packages/server-wallet/e2e-test/e2e-utils.ts

```diff
@@ -1,16 +1,36 @@
-import {spawn, ChildProcessWithoutNullStreams} from 'child_process';
+import {ChildProcessWithoutNullStreams, fork, spawn} from 'child_process';
+import {join} from 'path';
 
 import kill = require('tree-kill');
 import axios from 'axios';
 
 import Knex = require('knex');
 import {dbConfig} from '../src/db/config';
 
+import {PerformanceTimer} from './payer/timers';
+
 export type ReceiverServer = {
   url: string;
   server: ChildProcessWithoutNullStreams;
 };
 
+export const triggerPayments = async (
+  channelIds: string[],
+  numPayments?: number
+): Promise<void> => {
+  let args = ['start', '--database', 'payer', '--channels', ...channelIds];
+
+  if (numPayments) args = args.concat(['--numPayments', numPayments.toString()]);
+
+  const payerScript = fork(join(__dirname, './payer/index.ts'), args, {
+    execArgv: ['-r', 'ts-node/register'],
+  });
+  payerScript.on('message', message =>
+    console.log(PerformanceTimer.formatResults(JSON.parse(message as any)))
```

Comment:
Future work: in addition to logging to the console, store it as an artifact. They can then be tracked over time, or referenced eg. as a status check.